db/migrations/20220417031606_project_members_versions_customized/migration.sql

```diff
@@ -0,0 +1,156 @@
+    "ownerId" TEXT,
+    "name" TEXT NOT NULL,
+    "logo" TEXT,
+    "description" TEXT,
+    "valueStatement" TEXT,
+    "target" TEXT,
+    "demo" TEXT,
+    "repoUrl" TEXT,
+    "slackChannel" TEXT,
+    "isApproved" BOOLEAN NOT NULL DEFAULT false,
+    "status" TEXT DEFAULT 'Idea Submitted',
+    "categoryName" TEXT DEFAULT 'Experimenter',
+    "positions" TEXT,
+    "searchSkills" TEXT NOT NULL DEFAULT '',
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "membersCount" INTEGER NOT NULL DEFAULT 0,
+    "votesCount" INTEGER NOT NULL DEFAULT 0,
+    "isArchived" BOOLEAN NOT NULL DEFAULT false,
+    FOREIGN KEY ("ownerId") REFERENCES "Profiles" ("id") ON DELETE CASCADE ON UPDATE CASCADE,
+    FOREIGN KEY ("status") REFERENCES "ProjectStatus" ("name") ON DELETE SET NULL ON UPDATE CASCADE,
+    FOREIGN KEY ("categoryName") REFERENCES "Category" ("name") ON DELETE SET NULL ON UPDATE CASCADE
+);
+
+-- RedefineTables
+PRAGMA foreign_keys=OFF;
+CREATE TABLE "new_ProjectMembers" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "projectId" TEXT NOT NULL,
+    "profileId" TEXT NOT NULL,
+    "hoursPerWeek" INTEGER,
+    "role" TEXT,
+    "active" BOOLEAN NOT NULL DEFAULT true,
+    FOREIGN KEY ("profileId") REFERENCES "Profiles" ("id") ON DELETE CASCADE ON UPDATE CASCADE,
+    FOREIGN KEY ("projectId") REFERENCES "Projects" ("id") ON DELETE CASCADE ON UPDATE CASCADE
+);
+INSERT INTO "new_ProjectMembers" ("active", "hoursPerWeek", "id", "profileId", "projectId", "role") SELECT "active", "hoursPerWeek", "id", "profileId", "projectId", "role" FROM "ProjectMembers";
+DROP TABLE "ProjectMembers";
+ALTER TABLE "new_ProjectMembers" RENAME TO "ProjectMembers";
+CREATE INDEX "project_members_profile_id_idx" ON "ProjectMembers"("profileId");
+CREATE INDEX "project_members_project_id_idx" ON "ProjectMembers"("projectId");
+CREATE UNIQUE INDEX "project_members_project_id_profile_id_key" ON "ProjectMembers"("projectId", "profileId");
+PRAGMA foreign_key_check;
+PRAGMA foreign_keys=ON;
+
+-- CreateIndex
+CREATE INDEX "project_members_v_project_id_profile_id_key" ON "ProjectMembersVersions"("projectId", "profileId");
+
+-- CreateIndex
+CREATE INDEX "project_members_v_profile_id_idx" ON "ProjectMembersVersions"("profileId");
+
+-- CreateIndex
+CREATE INDEX "project_members_v_project_id_idx" ON "ProjectMembersVersions"("projectId");
+
+-- CreateIndex
+CREATE INDEX "projects_version_owner_id_idx" ON "ProjectsVersions"("ownerId");
+
+-- CreateIndex
+CREATE INDEX "projects_version_status_idx" ON "ProjectsVersions"("status");
+
+-- CreateIndex
+CREATE INDEX "projects_version_category_idx" ON "ProjectsVersions"("categoryName");
+
+-- CreateIndex
+CREATE INDEX "projects_version_isArchived_idx" ON "ProjectsVersions"("isArchived");
+
+--- Manually added for ProjectsVersions
```

Comment:
Here I start the manual modifications to the migrations file, to add triggers to auto populate version tables on insert/update/delete to the project and project members tables.